### DISCUSSION TITLE
Access the frozenInput in url()-callback or POST it

### DISCUSSION THREAD

**Discussion by nowrap** [digitalfortress-tech/typeahead-standalone#64] [Q&A] [answered]
Hello,

i am migrating a formular from jquery typeahead to typeahead-standalone in order to use a remote source.
Our api needs a generated signature for a request, so replacing the wildcard after the url()-callback doesn't work in our usecase.

Is there a way to get the frozenInput within the url()-callback?
Or is there an option to use the frozenInput as payload in a post request?

Regards
.nowrap

**Comment by niketpathak** [comment]
Hello, currently, the frozenInput is only available within the wildcard but it would be interesting to pass this info into the callback too to permit further customisation. Will include this in a future version.

**Reply by nowrap** [reply]
Thx alot ;)

**Reply by niketpathak** [reply, marked as answer]
A new version [`v4.23.0`](https://www.npmjs.com/package/typeahead-standalone) with the necessary changes was published.
